### ISSUE TITLE
TypeError: 'NoneType' exception when calling owner() method on a valid path

### ISSUE THREAD

**Issue by danj** [liormizr/s3path#13]
Calling bp.owner() I get this unexpected error

```
In [16]: bp.stat()
Out[16]: StatResult(size=4454, last_modified=datetime.datetime(2019, 2, 28, 19, 50, 20, tzinfo=tzutc()))

In [17]: bp.owner()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-17-c9b9b77d3c96> in <module>()
----> 1 bp.owner()

/usr/local/lib/python3.6/dist-packages/s3path.py in owner(self)
    411         if not self.is_file():
    412             return KeyError('file not found')
--> 413         return self._accessor.owner(self)
    414 
    415     def rename(self, target):

/usr/local/lib/python3.6/dist-packages/s3path.py in owner(self, path)
    142         key_name = str(path.key)
    143         object_summery = self.s3.ObjectSummary(bucket_name, key_name)
--> 144         return object_summery.owner['DisplayName']
    145 
    146     def rename(self, path, target):

TypeError: 'NoneType' object is not subscriptable
```

**Comment by liormizr** [comment]
Fix in pull-request #14

**Comment by liormizr** [comment]
fixed and deploy version 0.1.06
Enjoy
